modules/installation/proc-deploy-rhdh-instance-eks.adoc

```diff
@@ -0,0 +1,155 @@
+// Module included in the following assemblies:
+//
+// * assemblies/assembly-install-rhdh-eks.adoc
+
+[id="proc-deploy-rhdh-instance-eks.adoc_{context}"]
+= Deploying the {product-short} instance on {eks-short} with the Operator
+
+.Prerequisites
+
+* A cluster administrator has installed the {product} Operator.
```

Comment:
[SME] can you approve the addition of this Prereq?

Reply:
I think that looks good